rust-ceramic/src/metrics.md

```diff
@@ -0,0 +1,10 @@
+# Metrics
+
+Available metrics are:
+- `ceramic_api_requests_total`: Total number of Ceramic API requests
+- `ceramic_one_info`: Information about the Ceramic node
+- `ceramic_store_key_insert_count_total`: Total number of keys inserted into the store
+- `ceramic_store_value_insert_count_total`: Total number of values inserted into the store
+- `ceramic_store_store_query_durations_[sum, buckets, count]`: Store query durations
+- `ceramic_api_request_durations_[sum, buckets, count]`: API request durations
+- `ceramic_api_api_info`: Information about the Ceramic API
```

Comment:
Should we note that there are many more metrics and encourage operators to explore them via the `/metrics` endpoint? We can highlight these as the important ones but this is not an exhaustive list.

Reply:
Yeah, like the "help" I'd love a way to generate the metrics for the docs.

I don't see a way outside parsing all the source files.

Do you have any ideas how to make a rust command, or entrypoint, that would print the available metrics?